### PR TITLE
[diskless] adding example and clarification

### PR DESCRIPTION
**Pull request by hmeAtos**
Hello BB'Team,
I add example and clarification which are reported by less experienced users.
Regards,
@hmeAtos

### CODE REVIEW

File: roles/addons/diskless/files/disklessset.py
Comment (btravouillon) [inline roles/addons/diskless/files/disklessset.py:332]: 5000M != 5G here.
5G = 5120M

Comment (hmeAtos) [reply roles/addons/diskless/files/disklessset.py:332]: `        print('(Example: 5120M or 5G)')`

**Comment by hmeAtos** [comment]
Good ideo @btravouillon 
Done.

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:128]: 1.2Gb = 150MB.
In my tests yesterday, I was not able to get a 1.2GB file. It's either a 811MB squashfs or a 2.1GB rootfs image.

Comment (hmeAtos) [reply roles/addons/diskless/readme.rst:128]: 1.2Gb = 150MB ?? 
Did you check the size of the image in the booted node?
with `free -h` command

Comment (btravouillon) [reply roles/addons/diskless/readme.rst:128]: 1 Byte = 8 bits.
No I did not boot the diskless image. However I expect the total size to be 2.1GB.

Comment (btravouillon) [reply roles/addons/diskless/readme.rst:128]: I destroyed the test env... I will create a new image and check the memory consumption.

Comment (hmeAtos) [reply roles/addons/diskless/readme.rst:128]: 1.2 GB well seen ;)

Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: > I destroyed the test env... I will create a new image and check the memory consumption.

ok keep me informed
Thanks

Comment (btravouillon) [reply roles/addons/diskless/readme.rst]: Image:
```
  Image name: 		test
  Kernel version: 	vmlinuz-4.18.0-193.6.3.el8_2.x86_64
  Root password: 	root
  Image profile: 	1
  Image size: 		2100M
  SSH pubkey: 		/root/.ssh/id_rsa.pub
  Enable SELinux: 	y
```

Usage:
```
[root@c001 ~]# free -h
              total        used        free      shared  buff/cache   available
Mem:          1.9Gi       125Mi       877Mi       8.0Mi       982Mi       890Mi
Swap:            0B          0B          0B
[root@c001 ~]# df -h .
Filesystem           Size  Used Avail Use% Mounted on
/dev/mapper/live-rw  2.1G  1.4G  661M  69% /
```

Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]:
```
[root@pm5-nod40 ~]# free -h
              total        used        free      shared  buff/cache   available
Mem:           31Gi       **1.9Gi**        26Gi       121Mi       2.3Gi        27Gi
Swap:            0B          0B          0B
```

```
[root@pm5-nod40 ~]# df -h
Filesystem           Size  Used Avail Use% Mounted on
devtmpfs              16G     0   16G   0% /dev
tmpfs                 16G     0   16G   0% /dev/shm
tmpfs                 16G  122M   16G   1% /run
/dev/mapper/live-rw  **5.0G**  2.5G  2.6G  49% /
tmpfs                 16G     0   16G   0% /sys/fs/cgroup
tmpfs                3.2G     0  3.2G   0% /run/user/0
```

Comment (btravouillon) [reply roles/addons/diskless/readme.rst]: Any additional packages selected with the custom image option? Otherwise I would expect the same size than option 1.

Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: yes i have added additionnal packages and running the computes playbook.
All this added 500MB over the initial size.

Comment (btravouillon) [reply roles/addons/diskless/readme.rst]: OK. The size will depend on the list of additional packages, we cannot provide any hint here. Thanks for testing.

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:127]: [...] for the Standard or Custome **profile** is [...]

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:128]: Can you make a list in the same order than the option in the CLI?

 - 1.2GB for Standard...
 - 250 MB for Small
 - 130 MB for Minimal (typo: minimall)

No size for Custom since it is impossible to predict the final size.

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:149]: 5G is more reader-friendly

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:260]: Can't we consider a nicer command? Does `dnf clean all --installroot /var/tmp/diskless/workdir/space_image/` work?

**Review by btravouillon — COMMENTED** [review]
Nice change. A few comments inline to open discussion.

File: roles/addons/diskless/readme.rst
Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: yes its works
Thanks

File: roles/addons/diskless/readme.rst
Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: ok

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:129]: a blank line is missing, [check the rendering](https://github.com/bluebanquise/bluebanquise/blob/3484624dba92a7bc08fafc1da65ee93a37f5e48c/roles/addons/diskless/readme.rst)

File: roles/addons/diskless/readme.rst
Comment (btravouillon) [inline roles/addons/diskless/readme.rst:138]:
```suggestion
* With option 4, specify the packages version. Example: kernel-modules-4.18.0-193.6.3 kernel-4.18.0-193.6.3 kernel-devel-4.18.0-193.6.3
```

**Comment by oxedions** [comment]
Hello
Is this PR now ok so I can review it ?

**Review by btravouillon — CHANGES_REQUESTED** [review]
There are 2 pending items:
 * 5120M is not a blocker to me, even if I still think 5G is more user-friendly.
 *  I can't approve until the rendering of the list in the rst file is fixed.

**Comment by hmeAtos** [comment]
> There are 2 pending items:
> 
> * 5120M is not a blocker to me, even if I still think 5G is more user-friendly.
> * I can't approve until the rendering of the list in the rst file is fixed.

I'm taking care of it.

File: roles/addons/diskless/readme.rst
Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: fixed

File: roles/addons/diskless/readme.rst
Comment (hmeAtos) [reply roles/addons/diskless/readme.rst]: fixed

**Review by btravouillon — APPROVED** [review]
Looks good to me.

**Comment by btravouillon** [comment]
16 commits: please squash before merge.

**Review by bouriquet — APPROVED** [review]

**Pull request merged**
